humanResearch.ttl

```diff
@@ -1,122 +1,171 @@
+.
+<http://sweetontology.net/humanResearch#AppliedResearch>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Research> ;
+.
+<http://sweetontology.net/humanResearch#Assessment>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Investigation> ;
+.
+<http://sweetontology.net/humanResearch#Campaign>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#ExperimentActivity> ;
+.
+<http://sweetontology.net/humanResearch#Correction>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#ExperimentActivity> ;
+.
+<http://sweetontology.net/humanResearch#Difference>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#ExperimentActivity> ;
+.
+<http://sweetontology.net/humanResearch#EndToEndEnvironment>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#ResearchSetting> ;
+  owl:disjointWith <http://sweetontology.net/humanResearch#LaboratoryEnvironment> ;
+.
+<http://sweetontology.net/humanResearch#Evidence>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Assessment> ;
+.
+<http://sweetontology.net/humanResearch#Experiment>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Investigation> ;
+.
+<http://sweetontology.net/humanResearch#ExperimentActivity>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Experiment> ;
+.
+<http://sweetontology.net/humanResearch#HypothesisTest>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#ExperimentActivity> ;
+.
+<http://sweetontology.net/humanResearch#Investigate>
+  rdf:type owl:Class ;
+.
+<http://sweetontology.net/humanResearch#Investigation>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Research> ;
+  owl:equivalentClass <http://sweetontology.net/humanResearch#Investigate> ;
+.
+<http://sweetontology.net/humanResearch#LaboratoryEnvironment>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#ResearchSetting> ;
+.
+<http://sweetontology.net/humanResearch#MissionTestedEnvironment>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#ResearchSetting> ;
+  owl:disjointWith <http://sweetontology.net/humanResearch#LaboratoryEnvironment> ;
+.
+<http://sweetontology.net/humanResearch#Monitor>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Research> ;
+  owl:equivalentClass <http://sweetontology.net/humanResearch#Monitoring> ;
+.
+<http://sweetontology.net/humanResearch#Monitoring>
+  rdf:type owl:Class ;
+.
+<http://sweetontology.net/humanResearch#Observation>
+  rdf:type owl:Class ;
+  rdfs:subClassOf <http://sweetontology.net/humanResearch#Investigation> ;
+  rdfs:subClassOf <http://www.w3.org/ns/sosa/Observation> ;
```

Comment:
Currently SOSA is being referenced three times in this file... I wonder if using the prefix would be more appropriate notation?

Reply:
Up to you, though in the long run I'm not sure if we would want these mappings in the SWEET files themselves anyway. Maybe pull them all out into an alignment graph. There are some inevitable external refs and therefore dependencies (on rdf, rdfs, owl) without which we can't even get started. But when it comes to the next layer out (skos, prov, org, foaf, ... sosa) it depends if you think they are essential or more informative. If the alignments are provided as separate graphs, people can choose whether or not to load them.

Reply:
I agree with Simon's point here. I think you would want them in the same graph (file) only if you think they are core to the meaning in that graph, and are unlikely to be impacted by external changes. (So, a definition annotation in the same graph; a subclass relation within SWEET in the same graph; maybe a truly core relation like foaf or dc in the same graph; but a mapping to other domain concepts, would be nice to have in a separate graph. (One for sosa, one for geoscienceontology.org, etc. would be really nice for users, it seems to me.)

Reply:
OK to leave this inline in SWEET on an interim basis, but there should be a plan on how to manage alignment axioms in a scalable way long term.